src/Extension/Converters/ScssLinterViolationReportConverter.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Extension\Converters;
+
+use SprykerSdk\Sdk\Core\Appplication\Violation\AbstractViolationConverter;
+use SprykerSdk\Sdk\Core\Domain\Entity\Violation\PackageViolationReport;
+use SprykerSdk\Sdk\Core\Domain\Entity\Violation\ViolationReport;
+use SprykerSdk\Sdk\Core\Domain\Entity\Violation\ViolationReportConverter;
+use SprykerSdk\SdkContracts\Violation\ViolationReportInterface;
+
+class ScssLinterViolationReportConverter extends AbstractViolationConverter
+{
+    /**
+     * @var string
+     */
+    protected string $fileName;
+
+    /**
+     * @var string
+     */
+    protected string $producer;
+
+    /**
+     * @param array $configuration
+     *
+     * @return void
+     */
+    public function configure(array $configuration): void
+    {
+        $this->fileName = $configuration['input_file'];
+        $this->producer = $configuration['producer'];
+    }
+
+    /**
+     * @return \SprykerSdk\SdkContracts\Violation\ViolationReportInterface|null
+     */
+    public function convert(): ?ViolationReportInterface
+    {
+        $projectDirectory = $this->settingRepository->findOneByPath('project_dir');
+        if (!$projectDirectory) {
+            return null;
+        }
+
+        $jsonReport = $this->readFile();
+        if (!$jsonReport) {
+            return null;
+        }
+
+        $report = json_decode($jsonReport, true);
+        $report = array_filter((array)$report, static fn ($file) => $file['errored']);
```

Comment:
Why do you need `static` here?
And please add type-hint for `$file`

Reply:
I put static here because $this is not needed inside of the function. As far as I understand, closures are bound to the current context, but static closures are not, which makes them simpler versions of default closures. And I use them wherever I don't need $this i.e. almost everywhere. Is it wrong?
Typehint added

Reply:
You are right, but I mean what is the value from the `static` you try to get?

Reply:
This is just a habit to put `static` to closures by default because it makes them behave more simple and predictable. I read somewhere that in some edge cases, it may save an arbitrary amount of memory by not holding the reference to the object and allowing GC to collect it. But it is not the case here, so I can delete `static` if you insist.

Reply:
Let's do not premature optimization
`Static` doesn't change the behavior of this function
So please delete `static`